game_catalog/tests/test_views.py

```diff
@@ -0,0 +1,74 @@
+from rest_framework.test import APIClient
+import pytest
+
+pytestmark = pytest.mark.django_db
+
+
+@pytest.fixture
+def api_client():
+    return APIClient()
+
+
+@pytest.fixture
+def paths(game, genre):
+    return {
+        "genre_list": "/api/genres/",
+        "game_detail": f"/api/games/{game.id}/",
+        "toggle_favorite": f"/api/games/{game.id}/toggle_favorite/",
+    }
```

Comment:
QA should test this way, you should import paths from urls

Reply:
With routers usage we can only make it this way i think:
@pytest.fixture
def paths(game, genre):
    return {
        "genre_list": reverse("genre-list"),
        "game_detail": reverse("game-detail", kwargs={"pk": game.id}),
        "toggle_favorite": reverse("game-toggle-favorite", kwargs={"pk": game.id}),
    }
  will it be okay?

Reply:
Done

2185bc4a67d0084fc65ab40d5f2e194e1213a96c